File: python/programmers/level 1/약수의 합.py
def solution(n):
    약수 = []
    for i in range(1, int(n ** 0.5) + 1):
        if n % i == 0:
            약수.append(i)
            약수.append(n // i)
    약수 = list(set(약수))
    answer = sum(약수)
    return answer

File: java/src/main/java/ps_traning/programmers/level_1/약수의_합.java
package ps_traning.programmers.level_1;

public class 약수의_합 {
    class Solution {
        public int solution(int n) {
            int answer = 0;
            for (int num = 1; num <= (int) Math.sqrt(n); num++) {
                if (n % num == 0) {
                    answer += num;
                    if (n / num != num) {
                        answer += n / num;
                    }
                }
            }
            return answer;
        }
    }
}
